packages/nc-gui-v2/components/cell/MultiSelect.vue

```diff
@@ -47,8 +46,8 @@ const selectedTitles = computed(() =>
     ? typeof modelValue === 'string'
       ? isMysql
         ? modelValue.split(',').sort((a, b) => {
-            const opa = options.value.find((el: SelectOptionType) => el.title === a)
```

Comment:
Interface `SelectOptionType` is available

https://github.com/nocodb/nocodb/blob/chore/type-issues/packages/nocodb-sdk/src/lib/Api.ts#L313

Reply:
I'll revert those changes then 😄